docs/operators/csvformatter.md

```diff
@@ -0,0 +1,135 @@
+CsvFormatter
+============
+
+## Operator Objective
+This operator receives a POJO ([Plain Old Java Object](https://en.wikipedia.org/wiki/Plain_Old_Java_Object)) as an incoming tuple, evaluates the fields based on the constraints defined in the schema and converts the data in 
+the incoming POJO to a custom delimited string and finally emits the delimited string.
+
+CsvFormatter supports schema definition as a JSON string. Couple of ways to define schema are as below:
+
+1. **JSON File Based** - Read the file having content stored in JSON format and use that to convert to a string. This can be done using SchemaUtils implementation. Mostly used and recommended approach.
```

Comment:
This can be highlighted in an example. We need not mention it here. The user just needs to set "schema" property as a String.  How the user chooses to do that ( either SchemaUtils or any other way ) is his/her decision.